src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/Extensions/SyntaxGeneratorExtensions.cs

```diff
@@ -235,9 +254,18 @@ public static ImmutableArray<SyntaxNode> GetSetAccessorStatements(
             return [generator.ExpressionStatement(expression)];
         }
 
-        return preferAutoProperties
-            ? default
-            : generator.CreateThrowNotImplementedStatementBlock(compilation);
+        if (preferAutoProperties)
+            return default;
+
+        // Forward from the explicit property we're creating to the existing property it conflicts with if possible.
+        if (conflictingProperty is { SetMethod: not null, Parameters.Length: 0 } &&
```

Comment:
>Parameters.Length [](http://example.com/codeflow?start=58&length=17)

For my learning, setter properties don't have a parameter corresponding to "value"?

Reply:
no.  that's likely a bug.

Reply:
oh.  this was correct, but could be simplified.  the *property* has no parameters.  the *set accessor* has a single parameter.  i updated things accordingly.